src/libs/PolicyUtils.ts

```diff
@@ -117,6 +117,16 @@ const isFreeGroupPolicy = (policy: OnyxEntry<Policy> | EmptyObject): boolean =>
 
 const isPolicyMember = (policyID: string, policies: OnyxCollection<Policy>): boolean => Object.values(policies ?? {}).some((policy) => policy?.id === policyID);
 
+const isPolicyPayer = (policy: OnyxEntry<Policy> | EmptyObject, session: OnyxEntry<Session>, isApproved: boolean, isManager: boolean, isAdmin: boolean): boolean => {
```

Comment:
Let's change the name to `isPayer` since it would be used for IOUs too to avoid a regression

```suggestion
const isPayer = (policy: OnyxEntry<Policy> | EmptyObject, session: OnyxEntry<Session>, isApproved: boolean, isManager: boolean, isAdmin: boolean): boolean => {
```